AccessCheckoutSDK/AccessCheckoutSDK/client/validation/AccessCheckoutValidationInitialiser.swift

```diff
@@ -1,45 +1,55 @@
 import UIKit
 
 /// Class that is responsible for initialising validation using a given `ValidationConfig`
-public struct AccessCheckoutValidationInitialiser {
+internal struct AccessCheckoutValidationInitialiser {
     private static var presenters = [Presenter]()
 
     private var configurationProvider: CardBrandsConfigurationProvider
-
     /**
      This initialiser should be used to create an instance of `AccessCheckoutValidationInitialiser`
      */
-    public init() {
+    internal init() {
         let restClient = RestClient<[CardBrandDto]>()
         let transformer = CardBrandDtoTransformer()
         let configurationFactory = CardBrandsConfigurationFactory(restClient, transformer)
         self.configurationProvider = CardBrandsConfigurationProvider(configurationFactory)
     }
 
-    init(_ cardBrandsConfigurationProvider: CardBrandsConfigurationProvider) {
+    internal init(_ cardBrandsConfigurationProvider: CardBrandsConfigurationProvider) {
         self.configurationProvider = cardBrandsConfigurationProvider
     }
 
     /**
      This function should be used to initialise the validation using a given `ValidationConfig` provided by the merchant
-     - Parameter validationConfig: `ValidationConfig` that represents the configuration that should be used to initialise the validation
+     - Parameter validationConfiguration: `ValidationConfig` that represents the configuration that should be used to initialise the validation
+     - Parameter accessCheckoutClient: `AccessCheckoutClient` instance to access checkoutId and base URL
      */
-    public func initialise(_ validationConfiguration: ValidationConfig) {
+    internal func initialise(
+        _ validationConfiguration: ValidationConfig, accessCheckoutClient: AccessCheckoutClient
```

Comment:
Sorry to be a pain, can we rather pass in the `checkoutId` and the `baseUrl`? This is to avoid a circular dependency between `AccessCheckoutValidationInitialiser` and `AccessCheckoutClient`